fvm/evm/stdlib/contract_test.go

```diff
@@ -340,6 +348,112 @@ func TestEVMEncodeABI(t *testing.T) {
 		encodedABI,
 		result,
 	)
+	assert.Equal(t, computation, uint(len(cdcBytes)))
+}
+
+func TestEVMEncodeABIComputation(t *testing.T) {
```

Comment:
it seems like both of the tests could be refactored into a table test, only providing the script and computation used as input since as far as I see everything else is the same. You could also have then couple of more tests with more complex data or empty data etc.

Reply:
I encountered an error when I tried to do that for the test on the possible errors. Something was off and the assertions were mixed up. I used nested `t.Run()`.